### DISCUSSION TITLE
How should get work?

### DISCUSSION THREAD

**Discussion by freesig** [holochain/holochain#532] [Ideas]
Lets figure it out :)

**Comment by maackle** [comment]
Two years later, and we've (mostly) figured it out. (Hi @freesig !)

**Reply by freesig** [reply]
Hey! I can't even remember what this was about?

**Reply by maackle** [reply]
Probably something to do with all the ins and outs of the Cascade, when you get from the cache vs the network, etc. I'm sure you still think about that all the time :)
